core-primitives/enclave-api/src/side_chain.rs

```diff
@@ -32,7 +32,9 @@ pub trait SideChain: Send + Sync + 'static {
 		nonce: u32,
 	) -> EnclaveResult<()>;
 
-	fn execute_trusted_operations(&self) -> EnclaveResult<()>;
+	fn execute_trusted_getters(&self) -> EnclaveResult<()>;
+
+	fn execute_trusted_calls(&self) -> EnclaveResult<()>;
```

Comment:
Divided the `execute_trusted_operations` function into `execute_trusted_calls` and `execute_trusted_getters` functions